ste/mgr-JobPartMgr.go

```diff
@@ -691,6 +691,33 @@ func (jpm *jobPartMgr) createPipelines(ctx context.Context, sourceBlobToken azbl
 			jpm.pacer,
 			jpm.jobMgr.HttpClient(),
 			jpm.jobMgr.PipelineNetworkStats())
+
+		// Just in case we need to talk to blob while we're at it
+		var blobCred = azblob.NewAnonymousCredential()
+		if credInfo.CredentialType == common.ECredentialType.SharedKey() {
+			lcm := common.GetLifecycleMgr()
+			var err error
+			// Convert the shared key credential to a blob credential & re-use it
+			blobCred, err = azblob.NewSharedKeyCredential(lcm.GetEnvironmentVariable(common.EEnvironmentVariable.AccountName()), lcm.GetEnvironmentVariable(common.EEnvironmentVariable.AccountKey()))
+			if err != nil {
+				jpm.Panic(fmt.Errorf("sanity check: failed to initialize shared key credential: %w", err))
+			}
+		} else if credInfo.CredentialType != common.ECredentialType.Anonymous() {
```

Comment:
is oauth not a valid scenario here? or is oauth passed a different way?

Reply:
So, SharedKey gets picked up and converted, rather than getting directly included in the CreateBlobCredential call. Because this is anything NOT sharedkey or anonymous auth, OAuth by nature gets picked up here.